liveprofiler/wsgi.py

```diff
@@ -4,6 +4,16 @@
 
 import logging.config
 logging.config.fileConfig(CONFIG)
+log = logging.getLogger(__name__)
 
 from liveprofiler import app
 application = app.make_app(CONFIG)
+
+from liveprofiler import collector
+import uwsgidecorators
+
+@uwsgidecorators.timer(2)
```

Comment:
Can we move the interval to the config? Also, a comment on the interval units would be good. Any reason for the above value?

If not, I'd suggest 1 minute as a sensible default.